src/st/entity_equip_item_drop.h

```diff
@@ -83,10 +82,21 @@ void EntityEquipItemDrop(Entity* self) {
         prim = &g_PrimBuf[primIndex];
         prim->tpage = 0x1A;
         prim->clut = i + 464;
+
+#ifdef VERSION_PSP
+        prim->u0 = prim->u2 = (i & 7) * 0x10 + 1;
+#else
         prim->u0 = prim->u2 = (u8)(i & 7) * 0x10 + 1;
+#endif
         prim->u1 = prim->u3 = prim->u0 + 0xE;
+
+#ifdef VERSION_PSP
+        prim->v0 = prim->v1 = (i & 0x18) * 2 + 0x81;
+#else
         prim->v0 = prim->v1 = (u8)(i & 0x18) * 2 + 0x81;
+#endif
         prim->v2 = prim->v3 = prim->v0 + 0xE;
+
```

Comment:
Couldn't find a way to remove these casts on PSX. If someone can figure it out I can incorporate, see

// PSX: https://decomp.me/scratch/DQZyZ
// PSP: https://decomp.me/scratch/h5Klh